wordpress-mobile/WordPress-Android
# Issue/10650 material me screen

Fixes #10650 

This PR is based on #11071 , which needs to be merged first.

This PR moves Me and all nested screens to Material Theme and adds Dark Theme support. There are some code changes for cosmetic reasons. The photo picker and ZenDesk styling is not a part of this PR.

Design changes are reviewed before the code review is requested.

While refactoring is not in the scope of this PR, since I touched those files, I cleaned up/optimized some of the redundant code as suggested by AS, so do not be alarmed by the size of the PR. I suggest turning on the "Hide whitespace changes" option during the review.

There is a huge amount of PRs in this feature that covers all the screens within the app, so I'm requesting reviews from a wide variety of people.

@mattmiklic I wasn't sure what to do with About screen, so I improvised a bit :)
![comparison_me](https://user-images.githubusercontent.com/728822/72697364-a1d8e980-3af4-11ea-985f-2a2fda899892.png)


To test:
- Navigate to Me screen and check that it and all the nested screens are visually and functionally good in both Dark and Light themes. Make sure to check the username changer screen too.

PR submission checklist:
